.github/workflows/test-bot.yml

```diff
@@ -0,0 +1,23 @@
+name: brew test-bot
+on:
+  pull_request:
+    paths:
+      - "Formula/govuk-connect.rb"
+  push:
+    paths:
+      - "Formula/govuk-connect.rb"
+jobs:
+  test-bot:
+    runs-on: macos-latest
+    steps:
+      - name: Set up Git repository
+        uses: actions/checkout@v2
+      - name: Run brew test-bot
+        run: |
+          set -e
+          brew update
+          HOMEBREW_TAP_DIR="/usr/local/Homebrew/Library/Taps/alphagov/homebrew-gds"
+          mkdir -p "$HOMEBREW_TAP_DIR"
+          rm -rf "$HOMEBREW_TAP_DIR"
+          ln -s "$PWD" "$HOMEBREW_TAP_DIR"
+          brew test-bot govuk-connect
```

Comment:
Does this need to specify govuk-connect? I'm a bit unsure about whether this file is supposed to only be for govuk-connect (and should probably have a more specific filename) or if it's intended to be what is used if we have an additional tool or are able to resolve the govuk-cli blocker.

Reply:
It's supposed to be for everything. But we can only do it for one thing.